docs/core/project-sdk/overview.md

```diff
@@ -0,0 +1,119 @@
+---
+title: .NET Core project SDK overview
+description: Learn about the .NET Core project SDKs.
+ms.date: 02/02/2020
+ms.topic: conceptual
+---
+# .NET Core project SDKs
+
+.NET Core projects are associated with a software development kit (SDK). Each project SDK is a set of MSBuild [targets](/visualstudio/msbuild/msbuild-targets) and associated [tasks](/visualstudio/msbuild/msbuild-tasks) that are responsible for compiling, packing, and publishing code.
+
+## Available SDKs
+
+The following SDKs are available for .NET Core:
+
+| ID | Description | Repo|
+| - | - | - |
+| `Microsoft.NET.Sdk` | The .NET Core SDK | https://github.com/dotnet/sdk |
+| `Microsoft.NET.Sdk.Web` | The .NET Core [Web SDK](/aspnet/core/razor-pages/web-sdk) | https://github.com/aspnet/websdk |
+| `Microsoft.NET.Sdk.Razor` | The .NET Core [Razor SDK](/aspnet/core/razor-pages/sdk) |
+| `Microsoft.NET.Sdk.Worker` | The .NET Core Worker Service SDK |
+| `Microsoft.NET.Sdk.WindowsDesktop` | The .NET Core WinForms and WPF SDK |
+
+The .NET Core SDK is the base SDK for .NET Core. The other SDKs depend on the .NET Core SDK. For example, the Web SDK depends on both the .NET Core SDK and the Razor SDK.
+
+## Project files
+
+.NET Core projects are based on the [MSBuild](/visualstudio/msbuild/msbuild) format. Project files, which have extensions like *.csproj* for C# projects and *.fsproj* for F# projects, are in XML format. The root element of an MSBuild project file is the [Project](/msbuild/project-element-msbuild) element. The `Project` element has an optional `Sdk` attribute that specifies which SDK (and version) to use. To use the .NET Core tools and build your code, set the `Sdk` attribute to one of the IDs in the [Available SDKs](#available-sdks) table.
+
+```xml
+<Project Sdk="Microsoft.NET.Sdk">
+  ...
+</Project>
+```
+
+Another way to specify the SDK is with the top-level [Sdk](/visualstudio/msbuild/sdk-element-msbuild) element:
+
+```xml
+<Project>
+  <Sdk Name="Microsoft.NET.Sdk" />
+  ...
+</Project>
+```
+
+Referencing an SDK in one of these ways greatly simplifies project files for .NET Core. While evaluating the project, MSBuild adds implicit imports for `Sdk.props` at the top of the project file and `Sdk.targets` at the bottom.
```

Comment:
I'm torn between wanting everyone to know this (so being glad it's here) and feeling like it's redundant with `/visualstudio/msbuild/how-to-use-project-sdk`. Since it's already here and correct, I lean toward "leave it".